### ISSUE TITLE
Crash: NullPointerException

### ISSUE THREAD

**Issue by marek22k** [yairm210/Unciv#9561]
**Platform:** Desktop
**Version:** 4.6.18-patch1 (Build 876)
**Rulesets:** [Civ V - Gods & Kings, UnCiv-Great-Farmer, Civ V - Vanilla, UnCiv-Logicians]
**Last Screen:** `com.unciv.ui.screens.worldscreen.WorldScreen`

--------------------------------

OS: Linux (amd64, 6.1.0-1parrot1-amd64)
	Parrot OS 5.3 (Electro Ara)
Java: Eclipse Adoptium Temurin-11.0.19+7
	Max Memory: 1024 MB

--------------------------------

**Message:**
```
java.lang.NullPointerException
	at com.unciv.logic.civilization.Civilization.getEquivalentBuilding(Civilization.kt:497)
	at com.unciv.models.ruleset.Building$getRejectionReasons$1.invokeSuspend(Building.kt:540)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlin.sequences.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:129)
	at kotlin.sequences.SequencesKt___SequencesKt.none(_Sequences.kt:2130)
	at com.unciv.models.ruleset.Building.isBuildable(Building.kt:644)
	at com.unciv.logic.automation.city.ConstructionAutomation$filterBuildable$1.invoke(ConstructionAutomation.kt:70)
	at com.unciv.logic.automation.city.ConstructionAutomation$filterBuildable$1.invoke(ConstructionAutomation.kt:67)
	at kotlin.sequences.FilteringSequence$iterator$1.calcNext(Sequences.kt:171)
	at kotlin.sequences.FilteringSequence$iterator$1.hasNext(Sequences.kt:194)
	at com.unciv.logic.automation.city.ConstructionAutomation.addScienceBuildingChoice(ConstructionAutomation.kt:590)
	at com.unciv.logic.automation.city.ConstructionAutomation.chooseNextConstruction(ConstructionAutomation.kt:83)
	at com.unciv.logic.city.CityConstructions.chooseNextConstruction(CityConstructions.kt:631)
	at com.unciv.logic.civilization.Civilization.addCity(Civilization.kt:777)
	at com.unciv.ui.screens.worldscreen.unit.actions.UnitActions$getFoundCityAction$foundAction$1.invoke(UnitActions.kt:197)
	at com.unciv.ui.screens.worldscreen.unit.actions.UnitActions$getFoundCityAction$foundAction$1.invoke(UnitActions.kt:194)
	at com.unciv.ui.screens.worldscreen.unit.actions.UnitActions$getFoundCityAction$1.invoke(UnitActions.kt:220)
	at com.unciv.ui.screens.worldscreen.unit.actions.UnitActions$getFoundCityAction$1.invoke(UnitActions.kt:216)
	at com.unciv.ui.screens.worldscreen.unit.actions.UnitActionsTable$getUnitActionButton$1.invoke(UnitActionsTable.kt:64)
	at com.unciv.ui.screens.worldscreen.unit.actions.UnitActionsTable$getUnitActionButton$1.invoke(UnitActionsTable.kt:63)
	at com.unciv.ui.components.extensions.Scene2dExtensionsKt$onActivation$1.invoke(Scene2dExtensions.kt:194)
	at com.unciv.ui.components.extensions.Scene2dExtensionsKt$onActivation$1.invoke(Scene2dExtensions.kt:192)
	at com.unciv.ui.components.extensions.ActorAttachments.activate(Scene2dExtensions.kt:140)
	at com.unciv.ui.components.extensions.Scene2dExtensionsKt.activate(Scene2dExtensions.kt:183)
	at com.unciv.ui.components.extensions.ActorAttachments$addActivationAction$2.clicked(Scene2dExtensions.kt:151)
	at com.badlogic.gdx.scenes.scene2d.utils.ClickListener.touchUp(ClickListener.java:88)
	at com.badlogic.gdx.scenes.scene2d.InputListener.handle(InputListener.java:71)
	at com.badlogic.gdx.scenes.scene2d.Stage.touchUp(Stage.java:355)
	at com.unciv.ui.screens.basescreen.UncivStage.access$touchUp$s80204510(UncivStage.kt:17)
	at com.unciv.ui.screens.basescreen.UncivStage$touchUp$1.invoke(UncivStage.kt:78)
	at com.unciv.ui.screens.basescreen.UncivStage$touchUp$1.invoke(UncivStage.kt:78)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.screens.basescreen.UncivStage.touchUp(UncivStage.kt:78)
	at com.badlogic.gdx.InputEventQueue.drain(InputEventQueue.java:70)
	at com.badlogic.gdx.backends.lwjgl3.DefaultLwjgl3Input.update(DefaultLwjgl3Input.java:189)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:378)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:192)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:166)
	at com.unciv.app.desktop.DesktopLauncher.main(DesktopLauncher.kt:80)

```
**Save Mods:**
```
[Civ V - Gods & Kings, UnCiv-Logicians]
```

**Save Data:**
<details><summary>Show Saved Game</summary>

```
H4sIAAAAAAAAAO1dXZPbRpL8KxN8uosTNwYfJAFF3IOkXa10J1k6S7YfNvSAIaEZhEliDiTHmnX4vx9AAiA4RAFdmQ2eN+ywQ9KQA3RWoVFdVVld/evoIc42SboePf91tN6tbuJs9Nx7NppncbSNFz8l27vim238bTt6PvL/Mv2LE4zvo+38zhk9qy8IZtPffssvSh6SZfLPaJvfbzN6/o9fi0++i1ZxfunLKLuJsiTKv3iW325e3Da/5D5dJvMk3hx+yn/9VX7pNtvNy3sUn/7vLt5s30fr6LYYq/gki5fJbf4LJx/epstFvH5xW93sthDhY5zeL+PDB/HmPr8mv+Dksodkvk2zx9P775L15uSTbZQjmkfLF2/Le327X6ZZvPg+vj0oL5f/19H9MnqMs8+P94XEb3araD161tDBu/Q2F/ZEBfu/N9/HmzjK5nfxItfa6MVtlsx3y+0ui0dfTnU02tylu+Xiw328/lh8+vgxmf9cYMxVFv+uFXifpd+SVbJ9LKT4fBdfvY8eo1wn36XrOFfHi10OOml88GO0nqe7h+Ly6qO/rW+X0Xpx/CAfIFk2rnmZ5VNvs4weGp+9iZebZP1zcvzk3V20afzCp8fFOn6sfm551PNku1d+Pp2X6Xw/uQsZvo2ej91no/zKsZ//VlLgChb+11noeuOJ/9Uf+0F4M76J469j33Xd2XweTG5it3htGtPh6s3uJv/oa7pbL5L17avk4clEuU/vd8tq1AOcx6fPeHSzS5bbl/kfxT0KrKOP0TKaH+ZPPlfzW9U3qJ79k2dazvNkWcp6Kt/hZ+/w86T62Tn87FU/u0++d59877Tfz//ty7PRL2n2c7z43Db+pPg+2XzIctzraPkquk+20bKc9F+qJ/TqYLJGz52W13O0Te/fxV+35ZObHG7s5BLfpNttuvo+ub2rvqxgOteFUctny9/yafjhl3W8aEfwrGnnqkmKvuH/Oi/xcZLK7067mTxqqzYEf6rLRF2VlfxTWybaaiwHfyrMRGHHNfePqa91lMsULX9K14vcKS5EfRVnWXq1iK8+ptt0k4y+mCj1sEZ/2uar0cf8Pjnc5X7Q0essideL5ePoVO21i/LH1Lp9hZZ+4Z/qVKrzTbop/L8n2jz4y39MZZ5bhA/LxdXrKA+Lv+6WpDloV3cZj/wx9U1o822W7aOk3C0vYoKbY75hf9Ui+fp1r7Pidz9myXpeBH+F/t9H9wWcVXT/McryZ7DdP+n9B5+SfxZSj7JokezyD93rPE5JFkVOxPfz0CA+RA1uWMd07+P8N1eFKI0Myo9ViuU0gXL1b/uI7SqYTf89x7KJ98HLNJj67sxznDAIyoDsXbLZ7mOim2gTf86XpCgpbvb3LNps6mBj/+nruJiw+whq9CZZ7mdo/vq+3Mc4RfzzffJQpQvyb/L75g+tVGu63ibreJ2PdL3PZeTrXTPWdRoh0wmMj8v87yJSrQc6HcTK8IfRzwdv6sD4fu1yGKqzmpIvH4tPj5P4y4kC8nmByWpPym99dzKZMwM8Slz1+Y/pLpvvreRdHN8rtVGlE1x5Cnc9XMVIh4HOx4Gm2NErP1XBi+VulaxzY3P89MUq3RWYPOzdHl4xjqCY9wXu4p+dwjdGanul1fp+H2Wbu553uqHwt1m6blG2q3u7O4XvkLDWXs/70wD8Js028YaEfJwf5+MaI+8W2shstr7Q8oTtMm7qN7lTcmzCNuxRx5Jq+jDtGGavTtd2zK9B35UDgxStf1gn+2xs+st6zwA0MxNpmYT90PJd6YV9irfb5f6D+S7L8tu/Tx/iVVwiyL3QVdrwjFfll+/jVX7vMv38RDcH1Xi//XZwt1u/U1qf17kWc7+u294j69s5EhMznqXRolJpk384Ihhdn170bPTdPvoweoXU6tm/rc8G0JJDaanhgkTL7WhAvTmE3jQ6sfXKW1lteiXWexs9fkFDgldRYTa0XqXk00DOXiNnDa1UuHMNWBpLMRQuc8c7rFv47NzHo5yFk1xTMRuf5J573QmvP0JXC2McEuD22JPe0ldptNn2i+0fSfj2681il5ZbH+58fuMuJ9NovWrYnL8+xvrpKklsarSHWrI8QWEWl/qm6uK9i9epulXuTm6j7PGpQ9kU7alD2fyudCh/irIsSW06lMfair1LKTm+Spyvkyz+unxsw+lbwSl+e8Gp+Mrc6+x4dxGf8ff3xjmAiYI8xI4hbOSGuoUw8TJ9yDe052yOTfx1y86e3rNNVrkRqEzC6MV6nuT/utozNVg2tRuCNUfPqmszlhy1IaDbMkGyf2PixNjSnC89iApMY9L9dBctnzo3IkAglTdEcsDnnFFfbQMk5waLRf3e3IbCZYWz/ONJXQf6BMaHeRz1Z4sPl59fPUQk60tQLUUPailqUgIJCSAx2l5Z1GERbfdlnqYrqUGfzqpfwIETo+dYKRcVFl3vPHUgP5lS+UPVsvNdSuG5DntaF+MCMeWjMCDjXufesjt5vmwYE3xjkyx/F1x26ROzSCaRkBmrHkA2+TIOsuiXMW6e7OUZwm5b242MxtuH4g5ar8M0GkJiSE/0Uzp0bmtZm/R53Sb+1ABVFw1sqMc8EeyG9TTzRLJxGpxYatM4o5lDWbZMyxkmKhoxjqeH66fg9YfLz6+GMysTCY8mnDi/moqAZTBkwGALp9f3EGGcnu3n60pQTV3XAasVXQNhT97gfMS1VgGOpIC22tqzGqnBy267pW+xev+1W9/mHpwyAdwzATozF5ZKby29fGIsdpFa6XFveKbPqInO+utdptWOEWdkNyOAESSGKrBXiGtUgIdqRqIYT7ZJPuUYT74crLpu3FVeNzapr7McR3WEkYCzIPvGRBylqR1V52COXlfHAIjO28Ioi54EYL9bEWHlTcCy13C3jcu6e/Mjski69AgnD/AOycA7U1N48DrlgsKpsLj8Nd7E2VbwWZZpuri6P6gCVO35iGaR0FRwDGzqEiV9xrPD9TP9tHFabne42/nNuh4N5GBMJdiaKXR+tdUS1YmEURNAdzwYe7bD57TpXw6p1ztdL0G1egbTpztwAgJ14BWV4wBLN6ti6G5V6IM4dC413tmX0Tr/TyVQvySW6rrQF03MWGhLPe1nP6WcBTFHWxa19v1riq1raIwOGYieYJteYJjC355AG4jeht8zc1yqZNexZZwX21QlsBz32QpQJWqID1DFqAYDKoVkZubK2L+yVV4KezAzgFOUvIq6u5p2as84vnFG7IiSU5CsNChDOesPQM0iGzTKnBkYV9qCz/o4w2bON9ncGaJGFk6TJHiHCHINnAngc7/BesQ44+jVcXC4PrA2Idri58Mg52PU2kBEPr+d5tHYAjOVwGiSKLbATDjNTAQwyqonqJ2KL0FXR0h2PH+f0AUS+utGwkJur08mJH63dDtXwGace1Du5RMifmM7+Gl76PisTwJAb6fORqf39zoNtMIys2BiqkRFEFt5erKbrPU3xCISbQ0rui6LJfiXsMVSkb0mbdqTJADWBtM92UAAbGvrv1R/2F7NBQTAtiJ1icAze+XFiA4DI0WpZmCIJcNSSIhazsCAGJYhsgXOgfQQNeA19ei4fqW5r0Gq4HwFoB3Q5DfeBNplapcCro43UEXgytZrgcl2oxZF1NvbGi/Dh0Tp/csBqy58P+Pi7dTfBSA50rv1T5b7AkR3YJWfH4eH24Xg5D5cfn619WAv4HAGAk4qj3J+O40X0AHGclEEinNqVWmTvieoDTUmVuH53ATz0RdBiOUsdDlt5EgsKckDhORqHDrG0udIdMBtFRE0ciSWpHEEaQyzIR0oyA3DqKERU0gDFAxJqSQzpP2lN/aQSoU5ZkjFjUfMPiZ075fY3uFyCXKuC0ZX4yVbq7W0t1GzWHdvHn/a2zB3hNeLq/fx/mARwLNAozeRCrauU4kD10SZXekau+4tmlLA/G9LSQbU6Ie2kjjYZr+Qy+KEWHGF0OseVz6X4QmpDI/duoGQywiFF8wIhVxGKBQWJJP4rzHv87l0o68wCaWlWDl6tRRjrezachGh4GLoD0uB9542FIRVoCh8FVkFXbkdO6mxUHJje8NCGfW5Sy1339mfZfbl3CE5OdYLk6kTRo8Jcq7LReEaDWQON2i5Xr8g10qCljVUgrBXgCHKNAMSdSChBtLgSCvoGYl/BuJnsoIw2KkElsoLttxPU3vVBceW4vxepKdvcH2i6Jc+GfxeGcgCmrorfccQMvj+vOQUSSRam0KuKJwcTAjyQcGEK4ljWErqiPiJ/CD2fhvoUVEZUB3DrnnTYOjHRKKlPOtYSrSqcoV2qwbRxKXVfZLIYz0aITSjKbZ8stZrQWr9pMoU0klOi70PpRa+qpQievIG4JSgyU9xa8cASKWNKKqUoq1yNKmYSOPsI/nNAfYHSflA08hLFsISQDoqVHmoH6JNsrGo3jq2NRVCOrOlRnDepqjx1WAntpRyuN0HtqhA2u+kdApS+vICFYDVU0ctRHn9ubUzCa6MUseX7KNXqwO13uX1ulYzUKxZIUVzzOX1HWle60FdjdlablmThCwHN24RCUXRlYTE9obyFlhbCXnZPWYLLaXBK1HPgWpFVdeZm4raLBgr+x+q7Qqiko5Nh8Dr3zGZzyNPqJNDm8LqV8yuYUXL58ZOlQOBHcAy63F+/efdepFFw6feaw8L92ElEaxFuCEJMexFCOX9ub0ULZcbvidsadFMxI8fxDQzkQlK+KONJKb/z1qeiPj1rJqQk+8FMbH+VHxRKiAsRMXyJbGwRLzXL5NRFh6xbV6vKKz354rSQcsy+sxcSVAl23CRM6rFVdmUi7CkbJSJgNfyfkrHTk6dIXuOJEW3p2kHqVSEryIwuoMJESi1x/9o2VD2ReynSSRa0ccgNedUkRy2ysqlQ0FUBIWOcbG5r0Pq5KtiLTqySYAzilIo4obcS5ToSzuNVTTGBcrJpapfVXzX02IciTnxoNmCSy2UZUMTgQ7/TR1PAXT7Hmjl+RHHZAYnzPk80+yiNpIXSiJU4sHvgXRGtzVSYQDmQDypXCWzLVtfwUGtvSMdTmOfyBBP4TEkMnobCdtGCvg2rTlcqbUSl8OFRLLXY0rfI65DNxpnvi2JLYBBzbWmdFwing8wz1nn6vPhKOeqWK2VcpYq2Qz0fRGzijVDYClDMXY2UQvkEOAlolXyCV50y7RRd9kBJhXpCLRcb8LZYOQM6bxfAmtIQg1NnnTTjEKtigMRpaYlSw/KC3A39l6IGSnRqaV5sdtss0Rva6ZGckFZ6kGKVKe/C61NRK0J+VqSSNJUf5nKJPkkNdRzt6Tx1XDHCvq1blsPFvRF1RuQRgDZpQur7bYG7ZRycObI61WA/aIw2W9BzazoyCj5L+xJAOQXtI73QVSxWLDTY0QVmrAuWLkdQR4euS2Vf9+O3tpGV3GnmIoEA9k6umeG1P9LRYJhneFtcl8oiyee38dwXygRN5WSVRdyn6cUcyceEoLnpRBWVzodRUWZ6fZHgc3ApIa3Koasg9wbwthJ3WV0bJnNhIW080KVrlDul7LXql+iuEwTQBaciW82UlAWPGCrnKMrPVaVWL+bg7YqecjH9K9Fz7kkPef20XM0aeqSjJ10VKV9xk48bVOFVMfYod08xfNG0K7CjuXzcMWOoWSxZSuhIfUIEHJYIJnR39hgQM5GqiLUMVnc1iixaNL6PqDeziL9D7RzyUP5KKe3CJkEhi7GY6dKtMFLQZklQ8+JPMpALkYt13dRTbb9TAK/I+G/UGxak31k7KF8AhTZh063sBcpxOpdopVGIEHXJ4RQ2mcmiss57++jdbIslI+m6mcmr5CoA5rcnVJ6OVMHqoUpaUjscoM2JqqeTlVWM01+FyrzRZVxbBe5MviScqwHUbIXQpy80XW3pmaiJdxCb3jViM4NGrGNZX/lAsSu6OuYcnyXRw3jbTCCcLkGQgCimSw5mKAoJ9Y9FWMnZSemCtJ5Wcfxm8H6MB0binY2YlJgtF958gSj+K1huoRrewiTvuLJVNhZAxx3ipK/4gHrlnUtncSuIlUt1exbNBlTiiueMVzxM65RIsoQiydbDkauovnu3tMg1A6NdJaFjjjFE1hS0zcdTSoXaLU8uRfb9Mw/6EhOkdxWC9enS+7BXKH8UBilsLnOwcKKOhvLARy+VK9GCs8sj2BNzfY02aIvPJIr9fq40rYM4ZPdqW10qiUq2COJVq+PaIXgwGyqdPLeEGyqeGigCivaZBzCej6YIYMldfQAeV6xPwmzT7WVE5SqMEnmzai2tHOuNXmn7kNUOuSyz3XKcZ5qpnTE6yQwcnEkKuHlTQmmbGuV6IWtbZmSRa8/ykDa+5brtXwvWdrTcv0QNt5h0ToSWoxxrMla+QG0cmu9L1hF4nYAtU3iwscPGkjYiyKExZV5zHZO8e/pcnH1IYO3jKrVdJzrkqJs93AJEGXqtIjUZsxEDdLtMKQscwX+PIN7/GawLPOsegydSWYFRPtJ5lOI0pfUPge7G4M7wADcfPe7bJeAH77v/gR+XtYek+xR2RZWdL245q/DA2c9NtlhuwBhLDp7pg5vP3iAv4bg9OEgkoeMiuR6AHznEMHyN4hoza5/rs0qHNLJQWm7N94Os4MGRaNd+fRNyxSieBiniunsOIPQJtOJ8rFTJF0lueGoCFOKrJ2ZkLXsFvQZRbwGUj7VcKEKKApVPOrdcHTpnHodDWk3VHE7UghkRoMgTKWGljpSEs/CSTsQVRkwmKQbizSiaQ4QTz+SaUfxieuLEH1OieX1ci63N8PhG2OEJ4ovzHPT98QnmUd/YOaRYB19knX0BWOLu6w+STxKp+EKy+vbh+JeIEiYcZS62Q/BOIpd/FVYz92dOqXIbtwSO6wYEl1SiR+Y2BcLF+G9uaJrPoCRxPlC8VAdLalGriUWaD1ypYAD5iMC/A5V1hA2gGUyEL3+KANpggkEHovAkxBoyVGymwCBwOlFAPKLjoTMhF+0tsdVVgxHmwJi8ZyFxPDU+ymfEjz1F8O1rwxrRbe2rwzF5yDn9iiCt+PB2GEgAiOJ9JOgla2tTm7Vc7PYezckvz0TFWf7sIKZpAIhydguJ1LROzWaHACRiCnNLsGIzXjlaKJbYf1EC9n/0DdisNfwWXRqDBd02afhS+lFd8fQ4ZK9HYhjF30fJVcIH75CjI8PzIgss6OXOMeMIVMbNKFF4lLcba1jAy11Zyd3UKLRqBxPs5wmGepX6w7YoRhY42Bac9qXzOnR1JSiK2dcZqv0zFCmMZASf6r2qS7khqPZxpCkHkOKeqyi027PQ0HOdvF+8FohHqKGs0DSMW86JhBPekm9O3VMIJ44lPbrqRJeBBMp8XqmKT8820hmGek0Jzc+/sQnJKc4oWZsPT485SfCG2uaXpyQxN9EsGE29gVPSApQOp5ZogDbjwzRLnyOeMi0Cra8M9CCSmESUGpQoZxuqCvjSGVjbSQkVDHliAVwOoOI0m9Hi4i6uhUC3iaTRpkg8OTzrHQIePqNNMxEcZC8bcD0DlWyE7ZEZW4Tvf4oA2kLCQQ+i8BnEXgsAk9CoCUxYQSuhEBJFaLMRE2CyhJwJGiHZBitKT8sE1LJqJrk2i708GLIQ2Y22eUI9cKapFpabnv5nsA21FvT058eF+v40bzXi54L7FCZ3f2DNvRyVAfAFGKTo1/vHYeYd7KG8BlcNnQiru+Ga5u8vLe/yK0oZZYPXvXlRd9MMHHNVzJ59k4WEX0AJZkHOyHE+PijYESW6Us7L7zS0I4ZOrRB7w2weV+g/dDIR+5ePUSXX7G5tY7dg9vgkNWycrBvOD6ZbqjWZThhMuWYwRmX7yn9LfTygGQGA4oZDKVsoIplg5OJ1yTJV94Ats+OlF/W1eaStcFw7n0sHvumo8vwJJnUNFJHl7F1/mSSkKDrOPKL8KXYjB6dUuTGx5/4lKTrpiRdNyXpuin1xtbjw6/8lLJY9fiwyZPOXTa1uOK50arx4QXPEVusq8bH+TOx8YFq/A7+rMUrbj2a7YcsWiet9bTac5YdsTZLZ9Nwxk12QXUIeLNK2lWCcasQkJaVYNzkQ7t0CHi+jLSuRBmOvD3B9A5V+he2UGU+Fr3+KANpIwkEExbBhEXgswh8FoHHIvAkBFrWEUbgSghAbs7pBcSRiB1A2a3x1yJ0k6IgbiflcGKFolRIm9Juskr2Vzrqu3yZvdRhHqgnZtA7LTD2I4Bl1EogL1ecBOIqRksg7fEtkZ3v8a2/GG6P73HJbd3jK6/IZn1GLJKf2ANhNzbJDkHfLNOyZKLnoOQzTdcoJZ8J+xOsOyF6E0o+E72B6Eso2UvYmSHGxwdmRJb5WlMHkCOM5b2oqja5cCgl9wOGt9eIjX91TCHcwIzsasMW5Y7JsmA2STIm0zSVr4KmiUpnB74+IJnKgGIqQy7zWDr7cOLymku8ViEUbA0dLvFcRZ9kCS6ceK/CcbKTEsFVSseu6bhCtiMXWc5PcJXcRjWWeKvScnCCGX/x2EwknQrlxsef+IzkKmckVzkjucoZyVXOSK5yRnKVM5Kr5E4ArseHTT7XvLsen3z/4AXfEbseqMaHWzY4ZGWXI7pr+P561oWsMPFGkbSKBNNYISDtIsE0VghIy0gwjfJBWDoEPE9IWkeihkneqGJ6hyr9DVuYMs9NmzgcwYxFMGURTFkEExbBhEXgswh8FoHHIvAkBFq2FUbgSgg+rdNfThe79t7U+VROVvdZTWaMXqznSf6vq2yHnEjssAI5PQJhDCzcnfxagtPGpgqtKKYgwYqCDm2qsN9aU2Rlh2LNmCW2baqsZANyCWz3Li4fqr0TbdjlDm7CXtDtNl0rVScuPaqdJ4jmjUg8eDlg1yN2OWJXI3ExUpJ16A3EpUhJ1sFLBzE+PjAjMktPjhl+skHWkftV4evZgwLGZOdc9hz5ippDr2e71YzJfjls9fCYrF9msxpjMq9S+TYwNRhw1GBIUoMhRQ1WzjFusK85ctBhyUGHIwddlhx0OXLQY8lBjyMHydPDqgiebEJIkINcF0d2V1qVxSEzogQ5yFFthKfGpu7o3CE3Pv7EA5IcDEhyMCDJwYAkBwOSHAxIcjAgyUHubNt6fNjkc+216/HhJY9rPFCPT77/sMPjkMVUDumwHQ0QTAUGLBVYISBtIEEFVghIK0hQgRUC0g4SVGCFgLSEBBUoH5ilQ8ATeaQ1JNIf8k4a0ztUuXbYopTJdtqk4QgCFsGMRTBjEUxZBFMWwYRFMGER+CwCn0XgsQg8CQFeeuOymNw+TGZJBgKBIyGgGE0y64MrhLWYosGEJ0m/BaUoSlhTsl21sksSftFla8vhYo0wa4NZE8xaYNYAs/ZXNL9KAhC9gWhqlQQgbNqI8fGBGZFZynPMcJ4NApDc4EnuLyUISLLhLntWekUAotfTh0WQx1WwzXvGZPsgtqh4TJY1s7mUMZnNqXwpOB8VcvmsypvETfY1R0E6LAXpcBSky1KQLkdBeiwF6XEUpM9SkD5HQZLnmFVxO9n6i6Aguc6U7GazKndDZmIJCpIj9AhfkU3Y0RlDbnz8iYckBRmSFGRIUpAhSUGGJAUZkhRkSFKQ3Bm39fiwyee6f9fjw0se11GgHh9e8rmiqXp81v7gFCTrNFYIeBNI2kCCgqwQkFaQoCArBKQdJCjICgFpCQkKskIg32F7+KS5h+PtPDboYiieNqbFJt+BxcaTjqQNJdJG8hYj08RPGTm6qBUqbwBff3SESDtIIAhYBAGLYMYimLEIpiyCKYtgwiKYsAh8FoHPIvBYBB6LwGURuBICJVGLI3D6EOjpa9ZKOqyVZI0kayNZE8laSNZAsvaRNY+sdWSNI2sbWdPIWkbWMLJ2kTWLolXUcaq4ASLGxwdmRGZZ5DFDIzc4VXIfLrkNmOB0yaa/7NHtFaeKXl9xqnBYO+HCYvY4kopTRa9n2ySNyUZNbHX4mKxPZ5NTYzI9Vlfo4Tb7mmR1HZbVdThW12VZXZdjdT2W1fU4VtdnWV2fY3UnLKs74Vhd8ri7KqlAtlkjWF2uhye7S7BKLJHJbYLV5ThSwlsms4mwxFU6ExeZY3XrdCo3Pjzjq/HRG5TXw298NT56g/J62OJV48OnrHNnGtfjw8cucH3S6/HhTjJcK4h6fLiQlatDq8eHgz7SZTsaINYC8iaQtIG4418jIK0gHvrUCEg7iAd/NQLSEuLhb42AtIVEV6kKgX3utsJGG8qOO7DYaCOKbwcQ94ZpEZjdIVfJZhtl22R9+y6dR8WdCnV9yS++jVbxxyjL/8zVWRyqNrpfRo/7f/7j19H8Lt3E61fJQ37jd+ltMk+ifa/Sw+98frwvKJo3u1U+Yg6z/D8f7SGZF4MXv7B/Lp/2rU6Tr8k8v/jVbpk/sKjo6PjXdJWs93iKH5L7ZbrKfyp+6XOy2j/KVbrY3+GHAsX4iKHAXh4Q93EP5gnCQq63i/zDaXi9WLju1/FkGgdj/yaajm/8hTe+ntwE/jz0J5NrN//9+V08/3mzW+VX/PebHz4Em83dfzy6/uv4x1+yz98Wnz68eb35n9v/PCr2U6HQz7tsXWwg/D9SDRVv468BAA==
```
</details>

**Comment by marek22k** [comment]
It seems the crash is due `Requires a [Science] in at least [3] cities`  unique in a building.

**Comment by marek22k** [comment]
The example of the unique `Example: "Requires a [Culture] in all cities"` also created a crash.

**Comment by yairm210** [comment]
Well yeah, that's not a building name, makes sense
The mod checker should have said so as well

**Comment by marek22k** [comment]
But on https://yairm210.github.io/Unciv/Modders/Unique-parameters/#buildingfilter it also says that you can specify "Gold" or "Culture" and in the example it also says that.

**Comment by marek22k** [comment]
And no, the mod checker didn't say anything.
![Screenshot at 2023-06-10 22-24-36](https://github.com/yairm210/Unciv/assets/46723399/f8492b64-f105-4a5f-b2e3-927e7d04101e)

**Comment by Schoenios** [comment]
Is this a current problem, if so then I have it. I can't even start the game before it crashes.

**Comment by Schoenios** [comment]
> > Is this a current problem, if so then I have it. I can't even start the game before it crashes.
> 
> Yes you can. Just don't select any mod. 🥳 🤣 🙃 🤪 ... 🙊

No, it doesn't work at all. On my Android, when I click the game icon, I not only get a black screen for a millisecond, or instead I get the message "Unciv is repeatedly quitting". The problem is that I haven't even been able to get into the game since Sunday.

**Comment by Schoenios** [comment]
Hello Yarim,

Thank you for your feedback. Clearing memory from the app is out of the
question. An update that fixes the problem with "problem" mods would be
ideal.

I saw that you see "Per Root" or "Per ADB HACK" as an option. Not really
familiar with this, I watched a video ADB HACK to access the "mods folder".
My question would be, can one access the ADB without using windows? Anyway,
I'll figure out how to access the folder.

Sincerely, L. Schoenwaelder.

SomeTroglodyte ***@***.***> schrieb am Mo., 12. Juni 2023,
18:05:
